plugins/secretstores/googlecloud/googlecloud_test.go

```diff
@@ -0,0 +1,151 @@
+package googlecloud
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/auth"
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+func TestSampleConfig(t *testing.T) {
+	require.NoError(t, config.NewConfig().LoadConfigData(testutil.DefaultSampleConfig((&GoogleCloud{}).SampleConfig()), config.EmptySourcePath))
+}
+
+func TestInitSuccess(t *testing.T) {
+	plugin := &GoogleCloud{
+		STSAudience:        "https://localhost",
+		Log:                testutil.Logger{},
+		ServiceAccountFile: "./testdata/gdch.json",
+	}
+	err := plugin.Init()
+	require.NoError(t, err)
```

Comment:
Please avoid declarations wherever possible


```suggestion
	require.NoError(t, plugin.Init())
```

This helps avoiding side effects and duplicate checks.

Same for other instances below.